Revert default index attributes

donaldong opened the pull request:
This reverts commit 14522afa8b97ea160f361af9da625a2a06fafbd7 and #61 temporily.

This should be shipped together with the vacuuming feature.

katyho commented:
Oh I think you also might need to revert my change lol https://github.com/chanzuckerberg/redcord/pull/61

donaldong commented:
lol yeah, lets do that too.

donaldong commented:
update

:mag_right: [Click to see changes since last update](https://github.com/chanzuckerberg/redcord/compare/f6c4fc3cc87bb3dfc0d9eb63b1765693faed1b8c..607e3e85aca11e2568e62a609d93f5c30215030e).
